crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3612,6 +3612,10 @@ impl AuthorityPerEpochStore {
             all_certs,
             cancelled_txns,
         )?;
+        debug!(
+            "Assigned versions from consensus processing: {:?}",
```

Comment:
Is there tx_digest label in the tracing span?

Reply:
No because this is a map from all tx keys to the assignment in this commit